modules/ROOT/pages/connectivity-testing.adoc

```diff
@@ -0,0 +1,73 @@
+= Connectivity 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Test the connectivity defined for your module.
+
+== Before You Begin
+
+* Know how to define a connection for your module. For more information, refer to xref:connections.adoc[].
+* To create connectivity tests, add the `mtf-tools` dependency to your project:
++
+[source,xml,linenums]
+----
+<dependency>
+    <groupId>com.mulesoft.munit</groupId>
+    <artifactId>mtf-tools</artifactId>
+    <version>1.0.0</version>
+    <classifier>mule-plugin</classifier>
+    <scope>test</scope>
+</dependency>
+----
+
+CAUTION: Connectivity testing is available since Mule runtime engine 4.2.0. If your module has a minimum Mule version earlier than 4.2.0, add `minMuleVersion` to your connectivity testing suite, for example: `<munit:config name="connectivity-testing-suite" minMuleVersion="4.2.0"/>`.
+
+== Test Connectivity
+
+The `test-connectivity` processor establishes a connection using the given configuration. If testing connectivity with the given configuration fails, the processor fails with the proper exception, description, and error type (if present).
+
+=== Successful Connection
+
+When you test that a connection is valid, you are validating that the processor does not fail. A successful connection test looks as follows:
```

Comment:
I think "appears" might be better here. Look is more for what the actor does (I think).